ERCS/erc-7845.md

```diff
@@ -0,0 +1,644 @@
+---
+eip: 7845
+title: Minimal Orchestrator RPC
+description: A minimal data model that an Orchestrator can understand to help fulfil a request for a solution from a wallet or any other system.
+author: Kieran Goodary (@IAmKio), Pillar Wallet (@pillarwallet), Luke Wickens (@lbw33), Rana Khoury (@RanaBug)
+discussions-to: https://ethereum-magicians.org/t/erc-7845-minimal-orchestrator-rpc/21885
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-11-07
+---
+
+## Abstract
+
+> "Hey Google, swap my SHIB for Pillar"
```

Comment:
We like to avoid mentioning specific products in proposals. It unfairly advertises projects that exist today over ones that come after going final.

Reply:
Agreed - this has been changed. Thanks.